cmd/kube-controller-manager/app/core.go

```diff
@@ -59,7 +59,7 @@ import (
 )
 
 func startServiceController(ctx ControllerContext) (bool, error) {
-	serviceController, err := servicecontroller.New(
+	serviceController, err := servicecontroller.NewServiceController(
```

Comment:
`servicecontroller.New` is preferred, see https://github.com/golang/go/wiki/CodeReviewComments#package-names

Reply:
i see, but also many other controllers use long names.

Reply:
I'd suggest either leave it as is or change other controllers' `New...` function names

Reply:
OK. I see